.changeset/chatty-coats-wave.md

```diff
@@ -0,0 +1,7 @@
+---
+"@razorpay/blade": patch
+---
+
+fix(RadioGroup): use `radiogroup` as a11y role
+
+We've upgraded the RadioGroup component with 'radiogroup' role, significantly boosting accessibility for screen readers and assistive technologies.
```

Comment:
Let's communicate whats changed: 

```suggestion
Fixed accessibility of RadioGroup component by changing it's role to `radiogroup` instead of `group`
```